periodic/periodic.go

```diff
@@ -546,6 +556,16 @@ func formatDate(d *time.Time) string {
 		d.Hour(), d.Minute(), d.Second())
 }
 
+// Return a jitter time that is (+/-)10% of the duration t
```

Comment:
// addJitter returns a jitter time that is (+/-)10% of the duration t if t is >0